j2ee.core.utilities/test/unit/data/JavaApp/build.xml

```diff
@@ -7,9 +28,9 @@
     <import file="nbproject/build-impl.xml"/>
     <!--
 
-    There exist several targets which are by default empty and which can be 
-    used for execution of your tasks. These targets are usually executed 
-    before and after some main targets. They are: 
+    There exist several targets which are by default empty and which can be
+    used for execution of your tasks. These targets are usually executed
```

Comment:
These whitespace changes are not necessary. I generally prefer the patch to contain only the fix, not reformatting.